src/js/core.js

```diff
@@ -894,7 +898,7 @@ function MediumEditor(elements, options) {
         // http://stackoverflow.com/questions/17678843/cant-restore-selection-after-html-modify-even-if-its-the-same-html
         // Tim Down
         // TODO: move to selection.js and clean up old methods there
-        importSelection: function (inSelectionState) {
+        importSelection: function (inSelectionState, favorLaterSelectionAnchor) {
```

Comment:
Could you add as a comment an information about what should be `favorLaterSelectionAnchor`, why it is used, in which context and also the value it should receive?